### ISSUE TITLE
join when vac banned

### ISSUE THREAD

**Issue by Thyraxis** [explowz/cathook#36]
an option for bots to join casual matches while vac banned for an example

https://github.com/explowz/cathook/assets/132177323/2a255a49-66ca-4ca7-85ac-03833a8cfb68

that video was using
sv_cheats 1; fps_max 1; host_timescale 20
later i changed to
sv_cheats 1; fps_max 1; host_timescale 30

**Comment by Thyraxis** [comment]
yup it is

**Comment by Thyraxis** [comment]
it is a request. very helpful for hosting vac banned bots like me

**Comment by Thyraxis** [comment]
you can change how much host_timescale but i recommend 20 or 30
also the command should be executed on queue or when the casual game was found
`sv_cheats 1; fps_max 1; host_timescale 30`

**Comment by Thyraxis** [comment]
so i guess you can add that if you want

**Comment by Thyraxis** [comment]
nvm it actually still sets to 30

**Comment by Thyraxis** [comment]
fps_max.

**Comment by Thyraxis** [comment]
when sv_cheats was enabled. when you join a server with sv_cheats disabled it actually sets the sv_cheats disabled for the cilent which makes fps_max set to 30

**Comment by Thyraxis** [comment]
> What the fuck how is joining a Casual server while VAC banned even possible? Did you find an exploit or something to do such a thing? If so then oh boy get ready to maybe see a fuckton of Cheaters and bots join servers.

No. i think its been around since 5 or 4 years. i dont really remember

**Comment by qtTF2** [comment]
> What the fuck how is joining a Casual server while VAC banned even possible? Did you find an exploit or something to do such a thing? If so then oh boy get ready to maybe see a fuckton of Cheaters and bots join servers.

This is possible due to server latency and broken code in TF2 when asking about ur account status though steam.

The command to do this isnt really necessary as a VAC banned account u will get into a casual game thats far away from u.
